extension/basicauthextension/README.md

```diff
@@ -27,16 +27,17 @@ receivers:
 processors:
 
 exporters:
-  logging:
-    logLevel: debug
+  otlphttp:
```

Comment:
How about `otlp`? It should be used whenever possible instead of `otlphttp`, from what I understand.

Reply:
currently the intention is to support only http client